docs/docs/concepts/memory.md

```diff
@@ -14,15 +14,15 @@ Both are important to understand and implement for your application.
 
 ## Short-term memory
 
-Short-term memory lets your application remember previous interactions within a single [thread](persistence.md#threads) or conversation. A [thread](persistence.md#threads) organizes multiple interactions in a session, similar to the way an email or slack thread groups messages in a single conversation.
+Short-term memory lets your application remember previous interactions within a single [thread](persistence.md#threads) or conversation. A [thread](persistence.md#threads) organizes multiple interactions in a session, similar to the way email groups messages in a single conversation.
 
-LangGraph manages short-term memory as part of the agent's state, persisted via thread-scoped checkpoints. This state can normally includes the conversation history along with other stateful data, such as uploaded files, generated artifacts, and other results from side-effects. By storing these in the graph's state, the bot can access the full context for a given conversation while maintaining separation between different threads.
+LangGraph manages short-term memory as part of the agent's state, persisted via thread-scoped checkpoints. This state can normally include the conversation history along with other stateful data, such as uploaded files, generated artifacts, and other ??results from side-effects (too abstract)??. By storing these in the graph's state, the bot can access the full context for a given conversation while maintaining separation between different threads.
```

Comment:
Marked a sentence here as potentially too abstract

I am not sure what "results from side-effects" means in practice.

Don't merge without updating the copy

Reply:
ya maybe i'll just say "retrieved docs" or something